offload_vio/README.md

```diff
@@ -0,0 +1,22 @@
+In order to run the offloading plugins you need the following extra dependecies
```

Comment:
Move to `docs/` and add to `mkdocs.yml`.

Reply:
There was a time when we were aiming for per-component READMEs. For example, timewarp has one. Should we go with that approach or mkdocs? If we do mkdocs, we should probably make a hierarchy for the plugin documentation otherwise a flat navigation bar will be really hard to navigate.